pkg/descriptions.json

```diff
@@ -471,6 +471,18 @@
         ]
     },
     {
+        "Name": "System.Drawing.Primitives",
+        "Description": "Provides access to GDI+ basic graphics functionality that should be used in place of the corresponding types in System.Drawing.dll.",
```

Comment:
This is a weird description. There is no graphics functionality here, just plain, basic structures (which may be useful for graphics applications), definitely not tied to any particular API (GDI+). Also, these types aren't used "in place of" the types in System.Drawing.dll, they unify to them on .NET Framework.

Maybe something like this would be better:

"Exposes several primitive drawing structures, useful for representing geometric shapes."